src/Activity.js

```diff
@@ -52,15 +53,28 @@ class Activity {
       allData.flightsOfStairs += userDay.flightsOfStairs
       return allData
     }, {numSteps: 0, minutesActive: 0, flightsOfStairs: 0})
-    dayData.numSteps = Math.round(dayData.numSteps / allUsers.length)
-    dayData.minutesActive = Math.round(dayData.minutesActive / allUsers.length)
-    dayData.flightsOfStairs = Math.round(dayData.flightsOfStairs / allUsers.length)
+    Object.keys(dayData).forEach(key => dayData[key] = Math.round(dayData[key] / allUsers.length))
```

Comment:
Refactored this function to iterate through keys of the object and round each value accordingly.